.travis.yml

```diff
@@ -6,6 +6,7 @@ cache:
   directories:
     - $HOME/.composer/cache
     - downloads
+    - vendor
```

Comment:
this should speed-up tests by caching the composer vendor folder. only updated packages has to be downloaded.